src/SMBFile.h

```diff
@@ -82,8 +82,8 @@ class CSMBFile : public kodi::addon::CInstanceVFS, public P8PLATFORM::CMutex
 class ATTRIBUTE_HIDDEN CMyAddon : public kodi::addon::CAddonBase
 {
 public:
-  CMyAddon() { }
-  virtual ADDON_STATUS CreateInstance(int instanceType, std::string instanceID, KODI_HANDLE instance, KODI_HANDLE& addonInstance) override
+  CMyAddon() = default
```

Comment:
Ups, a small typo (`;` is missing).

After the Kodi request is in I make a check of addon in Windows UWP to confirm the right operation.